### PR TITLE
use latest version of nextcloud

### LINKED ISSUE #34: Please, integrate web-installer to deploy scripts
If you show me where is you "make release" scripts, maybe I can do it.

### PR DESCRIPTION
**Pull request by gratuxri**
Close https://github.com/nextcloud/web-installer/issues/34

### CODE REVIEW

**Comment by kesselb** [comment]
> Welcome to the Setup Wizard for **Nextcloud latest**!

Please remove `NC_VERSION` from `showWelcome` method. I still think we should stick to a [fixed major version](https://github.com/nextcloud/web-installer/issues/34#issuecomment-537150503) but fine by me.

**Comment by gratuxri** [comment]
NC_VERSION is removed now from showWelcome.

File: setup-nextcloud.php [hunk 1]
```diff
@@ -684,7 +684,7 @@ static public function showJsValidation(){
 	* Shows the welcome screen of the setup wizard
 	*/
 	static public function showWelcome() {
-		$txt='Welcome to the Setup Wizard for<br /><b>Nextcloud '.NC_VERSION.'</b>!<br /><br />This wizard will:<br />1. Check the server dependencies<br />2. Download Nextcloud<br />3. Install Nextcloud in a few simple steps';
+		$txt='Welcome to the Setup Wizard for<br /><b>Nextcloud </b>!<br /><br />This wizard will:<br />1. Check the server dependencies<br />2. Download Nextcloud<br />3. Install Nextcloud in a few simple steps';
```
Comment (kesselb) [inline setup-nextcloud.php]:
```suggestion
		$txt='Welcome to the Setup Wizard for<br /><b>Nextcloud</b>!<br /><br />This wizard will:<br />1. Check the server dependencies<br />2. Download Nextcloud<br />3. Install Nextcloud in a few simple steps';
```

**Comment by jospoortvliet** [comment]
I think this is a clever solution, far better than what we did before. @rullzer if you agree and the code looks good, let's please merge and deploy this!

**Review by ChristophWurst — APPROVED** [review]

**Comment by jospoortvliet** [comment]
@gratuxri thanks a lot for doing this!

**Pull request merged**